doc/configuration/introduction.md

```diff
@@ -12,9 +12,12 @@ log:
   format: json
 p2p:
   trusted_peers:
-    - "/ip4/104.24.28.11/tcp/8299"
-    - "/ip4/104.24.29.11/tcp/8299"
+    - address: "/ip4/104.24.28.11/tcp/8299"
+      id: ed25519_pk1tyz2drx5kvsp49fk0gunkl8ktlhjc976zj5erqwavnpfmrexxxnscxln75
+    - address: "/ip4/104.24.29.11/tcp/8299"
+      id: ed25519_pk13j4eata8e567xwdqp6wjeu8wa7dsut3kj0u3tgulrsmyvveq9qxqeqr3kc
   public_address: "/ip4/127.0.0.1/tcp/8080"
+  private_id: ed25519_sk1n649x7zrmt38q6dtqkvj769wru4vpkfnrppw6d83qd7jh3uhux7qwhg8q3
```

Comment:
Is this the private key? Considered a low-grade secret so it's OK to put it into the config file?